android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java

```diff
@@ -198,6 +215,11 @@ public void setBorderRadius(ButtonViewGroup view, float borderRadius) {
     view.setBorderRadius(borderRadius);
   }
 
+  @ReactProp(name = "rippleColor")
+  public void setRippleColor(ButtonViewGroup view, int rippleColor) {
```

Comment:
how do we handle the case when ripple color has been set and then reset to null? There is this `mUseCustomRippleColor` variable but we only set it to `true` and never to `false`